docs/getting_started/navigating_docs.md

```diff
@@ -0,0 +1,29 @@
+# Navigating CURC Documentation
+
+CU Research Computing (CURC) offers a variety of services encompassing High Performance Computing (HPC), cloud computing assistance, and storage solutions. Many of these services are available to users associated with CU Boulder, Anschutz Medical Campus (AMC), Colorado State University (CSU), and institutions who are members of the Rocky Mountain Advanced Computing Consortium (RMACC). Our team understands that navigating the various services we provide can be overwhelming for both new and experienced users. To aid users, we have constructed flowcharts that outline the core areas of our documentation that every user on our system should become familiar with. Please refer to the section [High Performance Computing Flowchart](#high-performance-computing-flowchart) for a diagram on how to navigate our HPC systems (clusters) and storage solutions. For a diagram on the various cloud computing resources we can assist you with, please refer to the [Cloud Computing Flowchart](#cloud-computing-flowchart) section. 
```

Comment:
Grammar Change "institutions who are members" to "institutions that are members"